movie-mlops-project/serving/services/data_service.py

```
@@ -0,0 +1,392 @@
+    service = DataService()
+    print("✅ 객체 생성 완료")
+    
+    # 2. 설정 확인
+    print(f"📁 CSV 경로: {service.csv_path}")
+    print(f"🏷️ 예상 컬럼 수: {len(service.expected_columns)}개")
+    
+    # 3. 파일 존재 확인
+    print("\n🔍 파일 존재 확인 중...")
+    result = service.check_file_exists()
+    
+    print(f"📊 결과:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)